### PR TITLE
[BEAM-2565] add integration test for conditional functions

### PR DESCRIPTION
**Pull request by xumingming**
@XuMingmin @jbonofre @takidau

### CODE REVIEW

**Comment by xumingming** [comment]
Retest this please

**Comment by xumingming** [comment]
Retest this please

File: dsls/sql/src/test/java/org/apache/beam/dsls/sql/integrationtest/BeamSqlConditionalFunctionsIntegrationTest.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
+package org.apache.beam.dsls.sql.integrationtest;
```
Comment (XuMingmin) [inline dsls/sql/src/test/java/org/apache/beam/dsls/sql/integrationtest/BeamSqlConditionalFunctionsIntegrationTest.java]: need to add license info

**Comment by xumingming** [comment]
Added license info.

**Comment by JingsongLi** [comment]
+R @JingsongLi, I'll take a quick look.

**Comment by JingsongLi** [comment]
Merged. Feel free to close.

**Comment by xumingming** [comment]
Thanks @JingsongLi @XuMingmin !

**Pull request closed without merging**